.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+### Introduction
+
+[KVIrc](https://github.com/kvirc/KVIrc) and related projects ([KVIrc website](https://github.com/kvirc/kvirc-www), [KVIrc themes](https://github.com/kvirc/kvirc-themes) and [KVIrc scripts](https://github.com/kvirc/kvirc-scripts)) now all use GitHub for development, i.e. for source maintenance, *pull requests* and the review of such code.   
+Not familiar with Git? Start by looking at GitHub's [collaborating pages](https://help.github.com/categories/collaborating/).
+
+The *pull request* process is important, especially the code submission part, KVIrc uses the automatic **travis** build system to compile the *pull request* and allows the person(s) involved the time to make any improvements or fix any issues arising before the **pr** is merged into the main source tree. 
+It also allows any interested **KVIrc development Team** members the opportunity to provide feedback and guidance where needed.
+
+###### General guidelines for creating pull requests:
+
+- **Create topic branches**. Don't ask us to pull from your master branch. 
+- **One pull request per feature**. If you want to do more than one thing, send multiple *pull requests*. 
+- **Send coherent history**. Make sure each individual commit message in your *pull
+  request* is meaningful.  
+  If you had to make multiple intermediate commits while developing, please squash them before sending them to us.  
+  In the end before merging you may be asked to squash your commit even some more.

```

Comment:
maybe `Before merging, you may be asked to squash your commits`. Since you can't squash a singular commits ;). Also since it's a bulleted list, it should probably be short and to the point anyways.